diary.md

```diff
@@ -0,0 +1,16 @@
+### Previous discussion
```

Comment:
would you say we should add some documentation here onto how and when to make these entries? E.g.: "After each round of collaboration in the merged simulation study, as outlined by the simulation structure, collaborators will document the process by making a diary entry. To facilitate this, a set of guiding questions is provided:"

Reply:
Yes! I will add it